WooCommerce/Classes/Analytics/WooAnalyticsEvent.swift

```diff
@@ -108,6 +109,14 @@ extension WooAnalyticsEvent {
     static func shippingLabelsAPIRequest(result: ShippingLabelsAPIRequestResult) -> WooAnalyticsEvent {
         WooAnalyticsEvent(statName: .shippingLabelsAPIRequest, properties: ["action": result.rawValue])
     }
+
+    static func ordersListLoaded(totalDuration: TimeInterval, pageNumber: Int, status: OrderStatus?) -> WooAnalyticsEvent {
+        WooAnalyticsEvent(statName: .ordersListLoaded, properties: [
+            "status": status?.slug ?? String(),
```

Comment:
[Q]: Any reason on why using `String()` rather than `""` ?

Reply:
I copy-pasted this from the previous code in `OrderListViewController`. So, no good reason. 😅 Is there any difference between using `String()` versus `""`?

Reply:
Not according to Apple!
<img width="744" alt="Screen Shot 2021-02-16 at 9 42 24 AM" src="https://user-images.githubusercontent.com/562080/108078157-52d4ef00-703b-11eb-94e4-2f8fc60717f1.png">